.stylelintrc

```diff
@@ -7,10 +7,13 @@
       "ignore": ["stylelint-commands", "after-comment"]
     } ],
     "declaration-colon-space-after": "always",
+    "indentation": [4, {
```

Comment:
Did we agree on 2 space indentation?

Reply:
I think we did agree to use indentation of 2

Reply:
ok